app/forms/state_file/nj_estimated_tax_payments_form.rb

```diff
@@ -1,10 +1,15 @@
 module StateFile
   class NjEstimatedTaxPaymentsForm < QuestionsForm
     set_attributes_for :intake,
-                       :estimated_tax_payments
+                       :has_estimated_payments,
+                       :estimated_tax_payments,
+                       :overpayments
 
+    validates :has_estimated_payments, presence: true
     validates_numericality_of :estimated_tax_payments, message: I18n.t("validators.not_a_number"), if: -> { estimated_tax_payments.present? }
-    validates :estimated_tax_payments, presence: true, allow_blank: true, numericality: { greater_than_or_equal_to: 0 }
+    validates_numericality_of :overpayments, message: I18n.t("validators.not_a_number"), if: -> { estimated_tax_payments.present? }
```

Comment:
why does this only need to be a number if they had estimated tax payments?

Reply:
If they answer `No` to the radio question, we don't need to validate anything about this field at all, because the user won't see it and doesn't fill it in. Behaves pretty similarly to Use Tax

Reply:
Sorry, no, I misread this. This is actually a typo. My bad! Will fix, thanks.